simphony/testing/abc_check_particles.py

```diff
@@ -393,7 +389,7 @@ def test_exeception_when_updating_bond_with_incorrect_id(self):
         # then
         with self.assertRaises(ValueError):
             # when
-            self.container.update_bond(bond)
+            self.container.update_bonds([bond])
 
     def test_remove_bond(self):
```

Comment:
I think that we should also rename the test method names to reflect the new method names